### PR TITLE
Add scenario for userprovided_tls_mtls_kafka-acls

### PR DESCRIPTION
**Pull request by rohit2b**
Added a new scenario scenario workflow document.

This is for the following security configurations:
- User provided TLS certificates
- Full network encryption
- mTLS authentication
- Kafka ACLs based authorization

### CODE REVIEW

File: security/userprovided-tls_mtls_kafka-acls/README.md [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+* Component TLS Certificates
+
+* Authentication credentials for Zookeeper, Kafka, Control Center, remaining CP components
+
+### Provide component TLS certificates
+
+Set the tutorial directory for this tutorial under the directory you downloaded the tutorial files:
+
+```
+export TUTORIAL_HOME=<Tutorial directory>/security/userprovided-tls_mtls_confluent_rbac
```
Comment (MosheBlumbergX) [inline security/userprovided-tls_mtls_kafka-acls/README.md:72]: should this be `export TUTORIAL_HOME=<Tutorial directory>/security/userprovided-tls_mtls_kafka-acls` ?

**Review by MosheBlumbergX — CHANGES_REQUESTED** [review]
in `component-certs/README.md` the first step is to 
```
openssl genrsa -out $TUTORIAL_HOME/generated/rootCAkey.pem 2048
```

But we don't have `generated` dir. 
we should add something like - 
```
mkdir $TUTORIAL_HOME/../../assets/certs/generated
```

like here - https://raw.githubusercontent.com/confluentinc/confluent-kubernetes-examples/master/security/production-secure-deploy/README.rst

**Review by MosheBlumbergX — CHANGES_REQUESTED** [review]
looks like a bug in the SR component: 

```
% k logs schemaregistry-0
[ERROR] 2021-07-29 16:10:38,214 [main] io.confluent.kafka.schemaregistry.rest.SchemaRegistryRestApplication initSchemaRegistry - Error starting the schema registry
io.confluent.kafka.schemaregistry.exceptions.SchemaRegistryInitializationException: Error initializing kafka store while initializing schema registry
        at io.confluent.kafka.schemaregistry.storage.KafkaSchemaRegistry.init(KafkaSchemaRegistry.java:297)
        at io.confluent.kafka.schemaregistry.rest.SchemaRegistryRestApplication.initSchemaRegistry(SchemaRegistryRestApplication.java:73)
        at io.confluent.kafka.schemaregistry.rest.SchemaRegistryRestApplication.configureBaseApplication(SchemaRegistryRestApplication.java:88)
        at io.confluent.rest.Application.configureHandler(Application.java:255)
        at io.confluent.rest.ApplicationServer.doStart(ApplicationServer.java:227)
        at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
        at io.confluent.kafka.schemaregistry.rest.SchemaRegistryMain.main(SchemaRegistryMain.java:43)
Caused by: io.confluent.kafka.schemaregistry.storage.exceptions.StoreInitializationException: Failed trying to create or validate schema topic configuration
        at io.confluent.kafka.schemaregistry.storage.KafkaStore.createOrVerifySchemaTopic(KafkaStore.java:181)
        at io.confluent.kafka.schemaregistry.storage.KafkaStore.init(KafkaStore.java:118)
        at io.confluent.kafka.schemaregistry.storage.KafkaSchemaRegistry.init(KafkaSchemaRegistry.java:295)
        ... 6 more
Caused by: java.util.concurrent.ExecutionException: org.apache.kafka.common.errors.TopicAuthorizationException: Authorization failed.
        at org.apache.kafka.common.internals.KafkaFutureImpl.wrapAndThrow(KafkaFutureImpl.java:45)
        at org.apache.kafka.common.internals.KafkaFutureImpl.access$000(KafkaFutureImpl.java:32)
        at org.apache.kafka.common.internals.KafkaFutureImpl$SingleWaiter.await(KafkaFutureImpl.java:104)
        at org.apache.kafka.common.internals.KafkaFutureImpl.get(KafkaFutureImpl.java:272)
        at io.confluent.kafka.schemaregistry.storage.KafkaStore.createSchemaTopic(KafkaStore.java:220)
        at io.confluent.kafka.schemaregistry.storage.KafkaStore.createOrVerifySchemaTopic(KafkaStore.java:173)
        ... 8 more
Caused by: org.apache.kafka.common.errors.TopicAuthorizationException: Authorization failed.



% k describe  pods schemaregistry-0 | grep -i ip
IP:           172.17.0.7
IPs:
  IP:           172.17.0.7
      HOST_IP:            (v1:status.hostIP)
      POD_IP:             (v1:status.podIP)
      HOST_IP:         (v1:status.hostIP)
      POD_IP:          (v1:status.podIP)
      /opt/confluentinc from cp-operator-scripts (rw)
  cp-operator-scripts:
mosheblumberg@C02F4360MD6T confluent-kubernetes-examples-mtls-acls % k logs kafka-0 | grep -i 172.17.0.7     
[INFO] 2021-07-29 16:10:38,143 [data-plane-kafka-request-handler-6] kafka.authorizer.logger logAuditMessage - Principal = User:r is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-cluster-rekey
```


Notice how the CN is `r` and not `sr`. 

```
% openssl x509 -in  $TUTORIAL_HOME/../../assets/certs/component-certs/generated/schemaregistry-server.pem -text -noout | grep -i DNS 
                DNS:sr, DNS:schemaregistry.mydomain.example, DNS:schemaregistry.confluent.svc.cluster.local
```

so must be the regex used on the broker?  @rohit2b

**Comment by MosheBlumbergX** [comment]
out of the box missing a bunch of acls, I wonder why not add the component users as super users? 

```
 % k logs kafka-2 | grep -i "is Denied Operation" | cut -d "=" -f 2- | sort -u 
 User:c3 is Denied Operation = ClusterAction from host = 172.17.0.12 on resource = Cluster:LITERAL:kafka-cluster
 User:c3 is Denied Operation = Delete from host = 172.17.0.12 on resource = Group:LITERAL:_confluent-controlcenter-0-command
 User:c3 is Denied Operation = Describe from host = 172.17.0.12 on resource = Group:LITERAL:ConfluentTelemetryReporterSampler--5277180018794338510
 User:c3 is Denied Operation = Describe from host = 172.17.0.12 on resource = Group:LITERAL:ConfluentTelemetryReporterSampler--7538903045944912799
 User:c3 is Denied Operation = Describe from host = 172.17.0.12 on resource = Topic:LITERAL:__consumer_offsets
 User:c3 is Denied Operation = Describe from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-ksql-confluent.ksqldb__command_topic
 User:c3 is Denied Operation = Describe from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-license
 User:c3 is Denied Operation = Describe from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-telemetry-metrics
 User:c3 is Denied Operation = Describe from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent_balancer_api_state
 User:c3 is Denied Operation = Describe from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent_balancer_broker_samples
 User:c3 is Denied Operation = Describe from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent_balancer_partition_samples
 User:c3 is Denied Operation = Describe from host = 172.17.0.12 on resource = Topic:LITERAL:_schemas_schemaregistry_confluent
 User:c3 is Denied Operation = Describe from host = 172.17.0.12 on resource = Topic:LITERAL:confluent.connect-configs
 User:c3 is Denied Operation = Describe from host = 172.17.0.12 on resource = Topic:LITERAL:confluent.connect-offsets
 User:c3 is Denied Operation = Describe from host = 172.17.0.12 on resource = Topic:LITERAL:confluent.connect-status
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-command
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-AlertHistoryStore-changelog
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-AlertHistoryStore-repartition
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-Group-ONE_MINUTE-changelog
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-Group-ONE_MINUTE-repartition
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-Group-THREE_HOURS-changelog
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-Group-THREE_HOURS-repartition
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-KSTREAM-OUTEROTHER-0000000106-store-changelog
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-KSTREAM-OUTEROTHER-0000000106-store-repartition
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-KSTREAM-OUTERTHIS-0000000105-store-changelog
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-KSTREAM-OUTERTHIS-0000000105-store-repartition
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-MetricsAggregateStore-changelog
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-MetricsAggregateStore-repartition
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-MonitoringMessageAggregatorWindows-ONE_MINUTE-changelog
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-MonitoringMessageAggregatorWindows-ONE_MINUTE-repartition
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-MonitoringMessageAggregatorWindows-THREE_HOURS-changelog
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-MonitoringMessageAggregatorWindows-THREE_HOURS-repartition
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-MonitoringStream-ONE_MINUTE-changelog
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-MonitoringStream-ONE_MINUTE-repartition
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-MonitoringStream-THREE_HOURS-changelog
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-MonitoringStream-THREE_HOURS-repartition
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-MonitoringTriggerStore-changelog
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-MonitoringTriggerStore-repartition
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-MonitoringVerifierStore-changelog
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-MonitoringVerifierStore-repartition
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-TriggerActionsStore-changelog
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-TriggerActionsStore-repartition
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-TriggerEventsStore-changelog
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-TriggerEventsStore-repartition
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-actual-group-consumption-rekey
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-aggregate-topic-partition-store-changelog
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-aggregate-topic-partition-store-repartition
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-aggregatedTopicPartitionTableWindows-ONE_MINUTE-changelog
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-aggregatedTopicPartitionTableWindows-ONE_MINUTE-repartition
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-aggregatedTopicPartitionTableWindows-THREE_HOURS-changelog
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-aggregatedTopicPartitionTableWindows-THREE_HOURS-repartition
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-cluster-rekey
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-expected-group-consumption-rekey
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-group-aggregate-store-ONE_MINUTE-changelog
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-group-aggregate-store-ONE_MINUTE-repartition
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-group-aggregate-store-THREE_HOURS-changelog
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-group-aggregate-store-THREE_HOURS-repartition
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-group-stream-extension-rekey
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-metrics-trigger-measurement-rekey
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-monitoring-aggregate-rekey-store-changelog
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-monitoring-aggregate-rekey-store-repartition
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-monitoring-message-rekey-store
 User:c3 is Denied Operation = DescribeConfigs from host = 172.17.0.12 on resource = Topic:LITERAL:_confluent-controlcenter-0-monitoring-trigger-event-rekey
 User:ksql is Denied Operation = Alter from host = 172.17.0.10 on resource = Topic:LITERAL:_confluent-ksql-confluent.ksqldb__command_topic
 User:ksql is Denied Operation = Alter from host = 172.17.0.11 on resource = Topic:LITERAL:_confluent-ksql-confluent.ksqldb__command_topic
 User:ksql is Denied Operation = AlterConfigs from host = 172.17.0.10 on resource = Topic:LITERAL:_confluent-ksql-confluent.ksqldb__command_topic
 User:ksql is Denied Operation = AlterConfigs from host = 172.17.0.11 on resource = Topic:LITERAL:_confluent-ksql-confluent.ksqldb__command_topic
 User:ksql is Denied Operation = Delete from host = 172.17.0.10 on resource = Topic:LITERAL:_confluent-ksql-confluent.ksqldb__command_topic
 User:ksql is Denied Operation = Delete from host = 172.17.0.11 on resource = Topic:LITERAL:_confluent-ksql-confluent.ksqldb__command_topic
 User:sr is Denied Operation = Create from host = 172.17.0.7 on resource = Cluster:LITERAL:kafka-cluster
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:__consumer_offsets
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-command
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-AlertHistoryStore-changelog
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-AlertHistoryStore-repartition
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-Group-ONE_MINUTE-changelog
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-Group-ONE_MINUTE-repartition
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-Group-THREE_HOURS-changelog
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-Group-THREE_HOURS-repartition
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-KSTREAM-OUTEROTHER-0000000106-store-changelog
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-KSTREAM-OUTEROTHER-0000000106-store-repartition
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-KSTREAM-OUTERTHIS-0000000105-store-changelog
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-KSTREAM-OUTERTHIS-0000000105-store-repartition
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-MetricsAggregateStore-changelog
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-MetricsAggregateStore-repartition
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-MonitoringMessageAggregatorWindows-ONE_MINUTE-changelog
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-MonitoringMessageAggregatorWindows-ONE_MINUTE-repartition
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-MonitoringMessageAggregatorWindows-THREE_HOURS-changelog
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-MonitoringMessageAggregatorWindows-THREE_HOURS-repartition
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-MonitoringStream-ONE_MINUTE-changelog
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-MonitoringStream-ONE_MINUTE-repartition
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-MonitoringStream-THREE_HOURS-changelog
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-MonitoringStream-THREE_HOURS-repartition
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-MonitoringTriggerStore-changelog
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-MonitoringTriggerStore-repartition
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-MonitoringVerifierStore-changelog
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-MonitoringVerifierStore-repartition
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-TriggerActionsStore-changelog
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-TriggerActionsStore-repartition
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-TriggerEventsStore-changelog
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-TriggerEventsStore-repartition
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-actual-group-consumption-rekey
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-aggregate-topic-partition-store-changelog
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-aggregate-topic-partition-store-repartition
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-aggregatedTopicPartitionTableWindows-ONE_MINUTE-changelog
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-aggregatedTopicPartitionTableWindows-ONE_MINUTE-repartition
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-aggregatedTopicPartitionTableWindows-THREE_HOURS-changelog
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-aggregatedTopicPartitionTableWindows-THREE_HOURS-repartition
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-cluster-rekey
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-expected-group-consumption-rekey
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-group-aggregate-store-ONE_MINUTE-changelog
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-group-aggregate-store-ONE_MINUTE-repartition
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-group-aggregate-store-THREE_HOURS-changelog
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-group-aggregate-store-THREE_HOURS-repartition
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-group-stream-extension-rekey
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-metrics-trigger-measurement-rekey
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-monitoring-aggregate-rekey-store-changelog
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-monitoring-aggregate-rekey-store-repartition
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-monitoring-message-rekey-store
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-controlcenter-0-monitoring-trigger-event-rekey
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-ksql-confluent.ksqldb__command_topic
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-metrics
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-monitoring
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent-telemetry-metrics
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent_balancer_api_state
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent_balancer_broker_samples
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:_confluent_balancer_partition_samples
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:confluent.connect-configs
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:confluent.connect-offsets
 User:sr is Denied Operation = Describe from host = 172.17.0.7 on resource = Topic:LITERAL:confluent.connect-status
 User:sr is Denied Operation = DescribeConfigs from host = 172.17.0.7 on resource = Topic:LITERAL:_schemas_schemaregistry_confluent
```

**Comment by rohit2b** [comment]
@MosheBlumbergX - I incorporated your suggestions, and tried with a new Regex. Could you review and test once on your end? Thanks!

File: security/userprovided-tls_mtls_kafka-acls/README.md [hunk 2]
```diff
@@ -0,0 +1,372 @@
...
+
+```
+kubectl get pods --namespace confluent
+```
+
+## Create TLS certificates
+
+In this scenario, you`ll configure authentication using the mTLS mechanism. With mTLS, Confluent components and clients use TLS certificates for authentication. The certificate has a CN that identifies the principal name.
+
+Each Confluent component service should have it`s own TLS certificate. In this scenario, you`ll
```
Comment (MosheBlumbergX) [inline security/userprovided-tls_mtls_kafka-acls/README.md:42]: NIT: readme format  - replace back tick with '

**Comment by MosheBlumbergX** [comment]
still missing ACLs on the scenario userprovided-tls_mtls_kafka-acls , added list 
[deniedop.log](https://github.com/confluentinc/confluent-kubernetes-examples/files/6999605/deniedop.log)

**Pull request merged**